_data/constants.yml

```diff
@@ -0,0 +1,8 @@
+raft_election_timeout: 2s
+raft_proposal_timeout: 2-4s
```

Comment:
The election timeout is 2-4 seconds, the proposal timeout is 3 seconds (in practice 3-6 seconds).

Reply:
This is now updated so that the variables in the file represent the various foo_ticks variables you mentioned in other comments, and the table in replication-layer.md now shows a value something like "multiply the ticks times the interval" to describe the value

in practice it means that eg the table on `replication-layer.md` shows a value for the raft election timeout like 'N ticks * M ms' rather than a range of say "2-4s" as shown here, followed by something in the description like

> Controlled by `COCKROACH_RAFT_REPROPOSAL_TIMEOUT_TICKS`, which is then multiplied by the default tick interval to determine the value. This value is then multiplied by a random factor to avoid election ties.